utilities/helperFunctions.js

```diff
@@ -78,8 +78,13 @@ const updateModMessage = (status, channel_id, text, user_id, ts, moderator) => {
     });
     return;
   }
-  
-  const emoji = status == "approved" ? ":heavy_check_mark:" : ":x:";
+
+  const emoji =
+    status == "approved"
+      ? ":heavy_check_mark:"
+      : status == "approved without at-channel"
+      ? ":heavy_minus_sign:"
+      : ":x:";
```

Comment:
This is quite the set of ternaries. While it is syntatically correct, do you think it would be more appropriate to use an `if-elseif-else` flow here?

Reply:
Yeah, but I didn't like how it looked 😂